### DISCUSSION TITLE
Selenide not installing the current chrome driver version to support browser version 116.0.5845.180

### DISCUSSION THREAD

**Discussion by PrateekNehraFL** [selenide/selenide#2468] [Q&A] [answered]
Can someone help to know how to fix the problem that most of the testers are facing as selenide installing 114 chromeDriver which is not supported for the latest chrome browser version 116.0.5845.180 , I was able to fix it locally by replacing the latest chromedriver . However, not sure how to fix that up while running the tests through pipeline on Ubuntu
![image](https://github.com/selenide/selenide/assets/82874048/d47e4dba-513a-44db-ad64-e40973668217)

**Comment by asolntsev** [comment, marked as answer]
Selenide 6.18.0 supports the latest Chrome 116 and automatically downloads chromedriver 116.
You don't need to configure anything for it.

But you probably have some old version of Selenium, Selenide or WebDriverManager in your project. 
Just run `gradle dependencies` or `mvn dependency:tree` - and you will see.

**Reply by PrateekNehraFL** [reply]
Thanks @asolntsev that helped after updating Selenide to 6.18 from 5.14 and Selenium to 4.12 from 3.14. However, now I ran into another problem after updating to Selenide 6.18 with Selenium 4.12 , put another one #2470 for the same.

**Comment by YevheniiPokhvalii** [comment]
But how to override the default Selenide behavior?
I'm trying to do it via Gradle and this parameter does not seem to do anything:
```
-Dwebdriver.chrome.driver=/usr/bin/chromedriver
```
Selenide still uses its own driver binary.

**Reply by asolntsev** [reply]
Your question is rather about using Gradle, not Selenide, right?
Do you run gradle with command like this? 
>  ./gradlew test -Dwebdriver.chrome.driver=/usr/bin/chromedriver

then you are passing the parameter to Gradle itself, not to tests.
To pass system property to tests, you need to write it in `build.gradle`. See for example:
https://github.com/selenide/selenide/blob/78fcf193a3d1a8643d49f73c2c89ff529d49abf5/gradle/tests.gradle#L129
